plugins/extend-give-wp/src/admin/views/featured_image_id_field.php

```diff
@@ -0,0 +1,16 @@
+<?php
+/**
+ *  View file for the featured image ID input/output field.
+ */
+?>
+<label>
+    <input id="featured-image-id" class="normal-text" name="extend-give-wp-options[featured-image-id]" type="number"
+           min="1" aria-describedby="featured-image-attachment-id"
+           value="<?php echo $attachment_id; ?>">
```

Comment:
Needs to be escaped with `echo esc_attr( $attachment_id );`.

Reply:
@hellofromtonya Good! I thought that the var had to be escaped. Just wasn't sure how to do that. That's done.